templates/views/layouts/default.html

```diff
@@ -89,24 +87,36 @@
                     </div>
                     <div class="col-lg-4 col-md-4 col-sm-4">
                         <p class="__footerp">About Us</p>
-                        <p class="__footera"><a href="">FAQ</a></p>
                         <p class="__footera"><a href="/team">Meet The Team</a></p>
                         <p class="__footera"><a href="">Join The Team</a></p>
-                        <p class="__footera"><a href="">Terms And Condition</a></p>
+                        <p class="__footera"><a href="">Community Guidelines</a></p>
                     </div>
                     <div class="col-lg-4 col-md-4 col-sm-4">
-                        <p class="__footerp">Membership</p>
-                        <p class="__footera"><a href="">Sign Up</a></p>
-                        <p class="__footera"><a href="">Sign In</a></p>
+                        <p class="__footerp">Community</p>
                         <p class="__footera"><a href="/events">Upcoming Events</a></p>
+                        <p class="__footera"><a href="https://bit.ly/joinscaslack">Join Our Slack Community</a></p>
+                        <p class="__footera">Follow us on social media</p>
+                        <div class="__social-media">
+                            <a href="https://twitter.com/SheCodeAfrica">
+                                <img src="/images/twitter.svg" alt="Twitter Logo" />
+                            </a>
+                            <a href="https://www.facebook.com/SheCodeAfrica/">
+                                <img src="/images/facebook.svg" alt="Facebook Logo" />
+                            </a>
+                            <a href="https://www.instagram.com/shecodeafrica/">
+                                <img src="/images/instagram.svg" alt="Instagram Logo" />
+                            </a>
+                        </div>
                     </div>
                 </div>
             </div>
         </div>
         <div class="__footerdownbg">
             <div class="container">
-                <p class="text-center"><a href="" target="_blank"> All right reserved. She code Africa. 2019</a></p>
-                <!-- <p class="text-center">Powered by <a href="http://keystonejs.com" target="_blank">KeystoneJS</a>.</p> -->
+                <p class="text-center">
+                    <a href="" target="_blank">
+                        All right reserved. She code Africa. 2020</a>
```

Comment:
Can the year here be dynamic? We won't have to keep updating it every year

Reply:
Yes, I've updated it now